tools/docker/tests/test_gw_repeater.sh

```diff
@@ -76,25 +76,25 @@ main() {
     dbg DELAY=$DELAY
 
     status "Start GW (Controller + local Agent)"
-    ${scriptdir}/../run.sh ${OPT} start-controller-agent -d -n ${GW_NAME} -m 00:11:22:33
+    ${scriptdir}/../run.sh ${OPT} start-controller-agent -d -n ${GW_NAME} -m 00:11:22:33 $@
     [ -z $GATEWAY_ONLY ] && {
         sleep ${DELAY}
         status "Start Repeater (Remote Agent)"
-        ${scriptdir}/../run.sh ${OPT} start-agent -d -n ${REPEATER_NAME} -m aa:bb:cc:dd
+        ${scriptdir}/../run.sh ${OPT} start-agent -d -n ${REPEATER_NAME} -m aa:bb:cc:dd $@
     }
     
     status "Delay ${DELAY} seconds..."
     sleep ${DELAY}
     
     error=0
     report "GW operational" \
-        ${scriptdir}/../test.sh ${OPT} -n ${GW_NAME}
+        ${scriptdir}/../test.sh ${OPT} -n ${GW_NAME} $@
```

Comment:
The options after `--` are options understood by run.sh. test.sh only understands the -n and -v options, and both of them are already passed, so I don't think it makes any sense to pass `$@` here. In fact, it will trigger errors from test.sh.